Makefile

```
@@ -85,3 +85,45 @@ test-all: test e2e ## Full gate + E2E
 
 bench: ## Run perf baseline against the Docker stack (override count with COUNT=500)
 	@bash testing/scripts/bench.sh $${COUNT:-100}
+
+test-extended-cleanup: ## Pre/post-RC cleanup — delete CLAUDE-TEST-* tickets + test mailbox messages
```

checkmake (0.3.2)

[warning] Target body for "test-extended-cleanup" exceeds allowed length of 5 lines (6).

(maxbodylength)